src/subscription_manager/printing_utils.py

```diff
@@ -125,7 +136,42 @@ def add_line():
     return '\n'.join(lines)
 
 
-def _none_wrap(template_str, *args):
+def _highlight_by_filter_string(template_str, *args, **kwargs):
```

Comment:
No particular reason for the leading _ here.

Also, I'd say name it 'highlight_by_filter_string_callback' just to make it clear it designed as a
callback method and not a general purpose method.

(I'd probably go for highlight_columnize_callback and none_wrap_columnize_callback and echo_columnize_callback, but I like (overly) long names...)